Directory.Build.props

```diff
@@ -19,8 +19,20 @@
     <LangVersion>9</LangVersion>
     <!-- Generate snupkg package -->
     <IncludeSymbols>true</IncludeSymbols>
+    <StrongNameKeyId>DotnetIot</StrongNameKeyId>
+    <SymbolPackageFormat>snupkg</SymbolPackageFormat>
   </PropertyGroup>
 
+  <Choose>
+    <When Condition="'$(StrongNameKeyId)' == 'DotnetIot'">
```

Comment:
Why do we need to condition this? Aren't these settings always used?

Reply:
we are reusing the properties that arcade uses in their strongname property and targets files, so in theory that key could still represent a different value which would be used at that point. This is mostly what I'm referring to: https://github.com/dotnet/arcade/blob/927f8d4d5036f68a5fc6d042f336bc9458027208/src/Microsoft.DotNet.Arcade.Sdk/tools/StrongName.targets#L24-L76

We could change it and not condition it and turn this into a traditional property group, but doing it like this it just makes it clear why the property StrongNameKeyId MUST be set, since if you don't set it and just set the publickeytoken and publickey, they will get overwritten later.

Reply:
A ok, makes sense. Maybe worth a comment?